controllers/taloscontrolplane_controller.go

```diff
@@ -240,6 +248,18 @@ func (r *TalosControlPlaneReconciler) Reconcile(req ctrl.Request) (res ctrl.Resu
 		return r.bootControlPlane(ctx, cluster, tcp, controlPlane, controlplaneNode)
 	// We are scaling down
 	case numMachines > desiredReplicas:
+		if err := r.etcdHealthcheck(ctx, cluster, ownedMachines); err != nil {
```

Comment:
do we need a similar check when scaling up?

probably scaling up is anyways fine, as they should join eventually (?)

Reply:
yep, that's what I observed. It only slows it down.